app/components/recruiter-conversations/conversation/status.js

```diff
@@ -2,16 +2,15 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
   profiles: Ember.inject.service(),
-  tagName: 'td',
-  classNames: ['grey'],
-  classNameBindings: ['unreadMessages'],
+  tagName: '',
```

Comment:
Ember wraps components in a div by default.  `tagName : ''` generates a component thats not wrapped in an html tag.